nuget/lib/dependabot/nuget/file_parser/project_file_parser.rb

```diff
@@ -290,29 +291,62 @@ def dependency_has_search_results?(dependency)
             dependency_urls = [UpdateChecker::RepositoryFinder.get_default_repository_details(dependency.name)]
           end
           dependency_urls_with_package = dependency_urls.select do |dependency_url|
-            response = execute_search_for_dependency_url(dependency_url)
-            next unless response.status == 200
+            dependency_url_has_matching_result?(dependency.name, dependency_url)
+          end
 
-            body = JSON.parse(response.body)
-            data = body["data"]
-            next unless data.length.positive?
+          dependency_urls_with_package.any?
```

Comment:
Since `dependency_urls_with_package` is just used for the return value, could we make the `select` above an `any?`?

Reply:
Done.